linters/javascript/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "rules": {
+    "consistent-return": 0,
+    "id-length": [2, {"exceptions": ["_", "x", "y"]}],
```

Comment:
My bad, this was extracted from a project. The only one char variable should be `_`. I'll remove the `x` and `y`.